src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -1044,6 +1044,10 @@ public List<StreamEntry> build(Object data) {
         String entryIdString = SafeEncoder.encode((byte[]) res.get(0));
         StreamEntryID entryID = new StreamEntryID(entryIdString);
         List<byte[]> hash = (List<byte[]>) res.get(1);
+        if (hash == null) {
+          responses.add(new StreamEntry(entryID, new HashMap<>()));
```

Comment:
We generally don't convert `nil` response from Redis to something else.
```suggestion
          responses.add(new StreamEntry(entryID, null));
```

Reply:
Fixed